src/main/java/com/efarm/efarmbackend/payload/request/auth/SignupFarmRequest.java

```diff
@@ -1,42 +1,45 @@
 package com.efarm.efarmbackend.payload.request.auth;
 
-import jakarta.validation.constraints.*;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
 import lombok.Getter;
 import lombok.Setter;
 
 @Setter
 @Getter
 public class SignupFarmRequest {
-    @NotBlank
-    @Size(min = 3, max = 30)
+    @NotBlank(message = "Imię nie może być puste")
+    @Size(min = 3, max = 30, message = "Imię musi mieć od 3 do 30 znaków")
     private String firstName;
 
-    @NotBlank
-    @Size(min = 3, max = 40)
+    @NotBlank(message = "Nazwisko nie może być puste")
+    @Size(min = 3, max = 40, message = "Nazwisko musi mieć od 3 do 40 znaków")
     private String lastName;
 
-    @NotBlank
-    @Size(min = 6, max = 30)
+    @NotBlank(message = "Nazwa użytkownika nie może być pusta")
+    @Size(min = 6, max = 30, message = "Nazwa użytkownika musi mieć od 6 do 30 znaków")
     private String username;
 
-    @NotBlank
-    @Size(max = 50)
-    @Email
+    @NotBlank(message = "Adres email nie może być pusty")
+    @Email(message = "Niepoprawny format adresu email")
+    @Size(max = 50, message = "Adres email może mieć maksymalnie 50 znaków")
     private String email;
 
-    @NotBlank
-    @Size(min = 6, max = 40)
+    @NotBlank(message = "Hasło nie może być puste")
+    @Size(min = 6, max = 40, message = "Hasło musi mieć od 6 do 40 znaków")
     private String password;
 
-    @Size(max = 12)
+    @Pattern(regexp = "^\\+?[0-9\\-]{9,12}$", message = "Niepoprawny format numeru telefonu")
+    @Size(max = 12, message = "Numer telefonu może mieć maksymalnie 12 znaków")
     private String phoneNumber;
```

Comment:
w tym momencie numer telefonu nie moze byc "" a jak gadalismy to formularze na froncie sa tak zrobione ze jak sie nie poda nr to przesle wlasnie pusty string